arangod/Pregel/Conductor/States/State.h

```diff
@@ -45,10 +45,7 @@ namespace conductor {
 
 struct State {
   virtual auto run() -> std::optional<std::unique_ptr<State>> = 0;
-  virtual auto receive(MessagePayload message)
-      -> std::optional<std::unique_ptr<State>> {
-    return std::nullopt;
-  }
+  virtual auto receive(MessagePayload message) -> void {}
```

Comment:
I was wondering whether it is reasonable to default-implement `receive` as empty? Shouldn´t this be virtual?